tasks/package.yml

```diff
@@ -5,12 +5,27 @@
     pkg: python-pycurl
     state: present
 
-- name: nodejs | Add the nodejs repository
-  apt_repository:
-    repo: ppa:chris-lea/node.js
+- name: node.js | get release
+  shell: lsb_release -c -s
+  register: distro
```

Comment:
perhaps you should add `changed_when: false` so that it passes the idempotence test when we get round to adding it :smile: